Documentation/getting-started-guide.md

```diff
@@ -9,19 +9,32 @@ If you're not on Linux, you should do all of this inside [the rkt Vagrant](https
 package main
 
 import (
-	"log"
-	"net/http"
+        "github.com/coreos/go-systemd/daemon"
+        "log"
+        "net"
+        "net/http"
 )
 
 func main() {
 	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		log.Printf("request from %v\n", r.RemoteAddr)
 		w.Write([]byte("hello\n"))
 	})
-	log.Fatal(http.ListenAndServe(":5000", nil))
+	ln, err := net.Listen("tcp", ":5000")
+	if err != nil {
+	fmt.Fprintf(os.Stderr, "%v\n", err)
```

Comment:
Why? It is one tab for `http.HnadleFunc` and 2 tabs for the body of its argument `func(...)`